api/interestr/api/tests.py

```diff
@@ -15,19 +15,19 @@
 # Create your tests here.
 class AuthTests(TestCase):
 
-    def setUp(self):
-        self.client = Client()
-        self.test_user = User.objects.create_user('name', 'wow@wow.com', 'wowpass123')
+	def setUp(self):
```

Comment:
lol, never trust the indentation based language...

Reply:
This is a "tabs vs. spaces" thing

Reply:
We can create a `.editorconfig` file to avoid this kind of errors, details: http://editorconfig.org/

Reply:
I think that we should better decide as a team whether to use tabs and spaces. When tabs and spaces are mixed it can get very annoying.